### PR TITLE
frontend/fix: Driver Info Card UI Fixes

### PR DESCRIPTION
**Pull request by PraveenGongada**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

Driver info card UI fixes

### CODE REVIEW

File: Frontend/android-native/mobility-customer/src/main/user/mobility_pm/res/drawable/ny_ic_wallet_filled.png
Comment (Vignesh-772) [inline Frontend/android-native/mobility-customer/src/main/user/mobility_pm/res/drawable/ny_ic_wallet_filled.png:1]: remove all the three images.

**Review by Vignesh-772 — CHANGES_REQUESTED** [review]

File: Frontend/ui-customer/src/Resources/Localizable/EN.purs [hunk 1]
```diff
@@ -369,11 +369,11 @@ getEN stringKey =
         PAY_VIA_CASH_OR_UPI -> "Pay via Cash / UPI"
         REQUEST_CALLBACK -> "Request Callback"
         CHOOSE_YOUR_RIDE -> "Choose Your Ride"
-        BOARD_THE_FIRST -> "Board the first "
+        BOARD_THE_FIRST -> "Board the first taxi from"
```
Comment (kavya-shree-s) [inline Frontend/ui-customer/src/Resources/Localizable/EN.purs:372]: Why taxi is hardcoded

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs [hunk 2]
```diff
@@ -520,9 +524,11 @@ rideInfoView push state =
           ][ rideInfoActionView push state 
            ]
          ]
-       , messageNotificationView push state
+       , if (not (os == "IOS" || state.props.enableChatWidget)) && ((any (_ == state.props.currentStage)) [ RideAccepted, ChatWithDriver]) then
```
Comment (kavya-shree-s) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs:527]: Add a comment mentioning its a temp fix

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs [hunk 3]
```diff
@@ -520,9 +524,11 @@ rideInfoView push state =
           ][ rideInfoActionView push state 
            ]
          ]
-       , messageNotificationView push state
+       , if (not (os == "IOS" || state.props.enableChatWidget)) && ((any (_ == state.props.currentStage)) [ RideAccepted, ChatWithDriver]) then 
+            linearLayout[height $ V 138, width MATCH_PARENT][]
+         else messageNotificationView push state
        , linearLayout
-        [ height $ V $ ((getInfoCardPeekHeight state) - if ((((any (_ == state.props.currentStage)) [ RideAccepted, ChatWithDriver]) && state.data.currentSearchResultType /= QUOTES && state.data.config.feature.enableChat) && state.data.config.feature.enableSuggestions) || (state.props.currentStage == RideStarted && os == "IOS") then 140 else 0)
+        [ height $ V $ ((getInfoCardPeekHeight state) - if isWidgetVisible then 140 else 0)
```
Comment (kavya-shree-s) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs:531]: Remove extra brackets

**Review by kavya-shree-s — APPROVED** [review]

**Pull request merged**